app/src/main/java/org/wikipedia/feed/configure/ConfigureFragment.kt

```diff
@@ -42,35 +49,6 @@ class ConfigureFragment : Fragment(), MenuProvider, ConfigureItemView.Callback {
 
         setupRecyclerView()
 
-        disposables.add(ServiceFactory.getRest(WikiSite("wikimedia.org")).feedAvailability
-            .subscribeOn(Schedulers.io())
-            .observeOn(AndroidSchedulers.mainThread())
-            .doAfterTerminate { prepareContentTypeList() }
```

Comment:
Notice that the old logic calls `prepareContentTypeList()` even in case of an error. The new logic only logs the error and doesn't do anything else (the progressBar remains spinning). Can we also call `prepareContentTypeList()` on error?

Reply:
Yes! I thought I added it 😅 Done!